examples/async_transcribe_media_from_url/async_transcribe_media_from_url.js

```diff
@@ -0,0 +1,60 @@
+const revai = require('revai-node-sdk');
+const fs = require('fs');
+const token = require('../config/config.json').access_token;
+
+(async () => {  
+    /* Initialize your client with your revai access token */
+    var client = new revai.RevAiApiClient(token);
+
+    /* Get account details */
+    var account = await client.getAccount();
+    console.log(`Account: ${account.email}`);
+    console.log(`Balance: ${account.balance_seconds} seconds`);
+
+    const jobOptions = {
+        metadata: "InternalOrderNumber=123456789",
+        callback_url: "https://jsonplaceholder.typicode.com/posts",
+        skip_diarization: true,
```

Comment:
should we add `skip_punctuation: false` here as well?

Reply:
skip_punctuation is not in the RevAiJobOptions interface, so I think no